### PR TITLE
Added new files in KB

### LINKED ISSUE #257: [KB] Add KB for google/clusterfuzzlite/actions/build_fuzzers
Knowledge Base is missing for google/clusterfuzzlite/actions/build_fuzzers.

### LINKED ISSUE #258: [KB] Add KB for google/clusterfuzzlite/actions/run_fuzzers
Knowledge Base is missing for google/clusterfuzzlite/actions/run_fuzzers.

### LINKED ISSUE #270: [KB] Add KB for lewagon/wait-on-check-action
Knowledge Base is missing for lewagon/wait-on-check-action.

### LINKED ISSUE #279: [KB] Add KB for actions/upload-release-asset
Knowledge Base is missing for actions/upload-release-asset.

### LINKED ISSUE #285: [KB] Add KB for softprops/turnstyle
Knowledge Base is missing for softprops/turnstyle.

### LINKED ISSUE #287: [KB] Add KB for kolonialno/mypy-action
Add KB for kolonialno/mypy-action

### LINKED ISSUE #288: [KB] Add KB for dustico/dusti-lock
Add KB for dustico/dusti-lock

### PR DESCRIPTION
**Pull request by h0x0er**
Fix #257
Fix #258
Fix #270
Fix #279 
Fix #285 
Fix #287
Fix #288

### CODE REVIEW

File: knowledge-base/google/clusterfuzzlite/actions/run_fuzzers/action-security.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: 'run-fuzzers'
+github-token:
+  action-input:
+    input: github-token #CHECKOUT: https://github.com/google/clusterfuzzlite/blob/main/actions/run_fuzzers/action.yml#L28
+  permissions:
+    actions: read
+    actions-reason: to fetch the artifacts #CHECKOUT: https://github.com/google/oss-fuzz/blob/0bf87a10c4c1df2514dd82023e4da2d2d26b2e1c/infra/cifuzz/filestore/github_actions/github_api.py#L47
```
Comment (varunsh-coder) [inline knowledge-base/google/clusterfuzzlite/actions/run_fuzzers/action-security.yml:7]: This also needs `contents: read` to clone the repo.

File: knowledge-base/google/clusterfuzzlite/actions/build_fuzzers/action-security.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: 'build-fuzzers'
+github-token:
+  action-input:
+    input: github-token #CHECKOUT: https://github.com/google/clusterfuzzlite/blob/main/actions/run_fuzzers/action.yml#L28
+  permissions:
+    actions: read
+    actions-reason: to fetch the artifacts #CHECKOUT: https://github.com/google/oss-fuzz/blob/0bf87a10c4c1df2514dd82023e4da2d2d26b2e1c/infra/cifuzz/filestore/github_actions/github_api.py#L47
```
Comment (varunsh-coder) [inline knowledge-base/google/clusterfuzzlite/actions/build_fuzzers/action-security.yml:7]: This also needs `contents: read` to clone the repo.

File: knowledge-base/google/clusterfuzzlite/actions/run_fuzzers/action-security.yml (hunk 1 above)
Comment (h0x0er) [reply knowledge-base/google/clusterfuzzlite/actions/run_fuzzers/action-security.yml:7]: @varunsh-coder I added the required `permission`

File: knowledge-base/google/clusterfuzzlite/actions/build_fuzzers/action-security.yml (hunk 2 above)
Comment (h0x0er) [reply knowledge-base/google/clusterfuzzlite/actions/build_fuzzers/action-security.yml:7]: @varunsh-coder I added the required `permission`

**Pull request merged**